### ISSUE TITLE
Make rich text fields more screen reader friendly

### ISSUE THREAD

**Issue by onlinedigitaltrade** [PrestaShop/PrestaShop#25466]
Is your feature request related to a problem?

Hi. I've been using Prestashop for over 7 months , and I've liked it so far. However I've found that rich text fields like the one for putting the product description and summary on the product page and the one where the content of cms pages go cannot be reached by screen readers. I have used JAWS for Windows and the Windows narrator but for some reason screen readers cannot reach the part where you input text for rich text fields on BO. Like all of the formatting tools in the text box are accessible, however screen readers cannot recognize the input field. They just can't be accessed when using the keyboard with a screen reader while navigating through the BO. So I can basically do all kinds of fancy things with the text like changing the font and font size, making the text bold or italic, but I can't input the text itself so that it shows up in the FO. 

#### Describe the solution you'd like

All text fields in the back office should be screen reader friendly, just like here on GitHub so that blind people or people with visual impairments can use prestashop to build their e-comerce websites. By the way I'm visually impaired and i've been able to build my shop pretty much by myself, so I look forward to keep using prestashop in the future.

#### Alternatives you've considered

I have found a way to work around this issue, although it is less than ideal. I learned that when using JAWS for Windows, I can disable the virtual cursor and navigate to the text box using the tab key. I can then paste text that I've copied from word or other program when JAWS announces the page title, since oddly it is when it reaches the input field on the text box. I know it is hard to explain, that's why this workaround is not ideal. I hope that you guys fix it in a future release like Ps 1.7.9 because it would be really helpful.

Additional context

By the way, I'm using Prestashop 1.7.7.5 with PHP 7.3.

**Comment by khouloudbelguith** [comment]
Hi @onlinedigitaltrade,

Thanks for your issue.
We are awre of this issue.
Your ticket is a duplicate of #23085
So, I close it.

Thanks!

**Comment by khouloudbelguith** [comment]
Duplicate of #23085
